httphandler/response_merger.go

```diff
@@ -85,6 +94,16 @@ func (rd *responseMerger) handleFailedResponces(
 	return alreadysent
 }
 
+func shouldBeFilteredInMaintenanceMode(failedTup transport.ResErrTuple) bool {
+	if failedTup.Req.Method == http.MethodPut || failedTup.Req.Method == http.MethodDelete {
```

Comment:
Are we sure that Post is not necessary here?

Reply:
As agreed we'll leave POSTs out for now